src/assets/simp-environment/environments/simp/hieradata/scenarios/poss.yaml

```diff
@@ -14,4 +14,4 @@
 # NOTE: Details for the 'poss' scenario can be found in the
 # 'simp::scenario::poss' class
 
-classes : []
+simp::scenario: 'poss'
```

Comment:
Not anymore. This was to make sure users didn't have to set simp::scenario and $simp_scenario at the same time in two different places.